packages/nx/src/project-graph/error-types.ts

```diff
@@ -36,12 +27,12 @@ export class ProjectGraphError extends Error {
       `Failed to process project graph. Run "nx reset" to fix this. Please report the issue if you keep seeing it.`
```

Comment:
Let's change this message while we are here. This currently treats `nx reset` as the primary/recommended resolution when it should only ever be the fallback/worst case.

This original error message was written when graph errors were anticipated to be far less frequent, but now with crystal plugins they are much more commonly expected because syntax errors/missing files etc from userland config files can prevent the graph from building.

```suggestion
      `Failed to process project graph, there may be an issue with one of your Nx plugins. If the cause is not obvious from the error message, running "nx reset" may fix it. Please report the issue if you keep seeing it.`
```

Reply:
That's only my suggestion off the top of my head, definitely open to futher refinement